syntheseus/cli/eval_single_step.py

```diff
@@ -397,21 +423,27 @@ def compute_metrics_from_config(
 
 
 def print_and_save(results: EvalResults, config: EvalConfig, suffix: str = "") -> None:
-    # Extract the top_k results for the chosen values of `k`; the other values are not printed.
-    chosen_topk_results = {x: results.top_k[x - 1] for x in config.print_idxs}
+    chosen_top_k_results: Dict[str, Dict[int, List[float]]] = {}
 
     # Print the results in a concise form.
     for f in fields(results):
-        if f.name not in ("model_info", "top_k", "predictions", "back_translation_predictions"):
-            print(f"{f.name}: {getattr(results, f.name)}")
+        if f.name not in ("model_info", "predictions", "back_translation_predictions"):
+            if f.name.endswith("top_k"):
+                top_k = getattr(results, f.name)
+                if top_k is not None:
+                    # Extract the top-k for the chosen values of `k`; other values are not printed.
+                    chosen_top_k_results[f.name] = {x: top_k[x - 1] for x in config.print_idxs}
+            else:
+                print(f"{f.name}: {getattr(results, f.name)}")
```

Comment:
Just to be clear, this part of the PR is unrelated to stereochemistry, right? It just seems to neaten up the printing.

Reply:
Slightly related: this PR adds `match_modulo_stereo_top_k`, which (similarly to the pre-existing `back_translation_top_k`) would be printed verbosely on the CLI instead of limiting to just 1/3/5/10/20/50. I tidied up the printing in a general way which also fixes this oversight for `back_translation_top_k`.